views/partials/email_partials/email_single_breach.hbs

```diff
@@ -3,9 +3,9 @@
       <td class="email-breach-image" valign="top" style="padding-bottom: 10px; width: 40px;">
         <img class="breach-logo" 
           {{#if breachAlert }}
-            src="{{ SERVER_URL }}/img/logos/missing-logo-icon.png"
+            src="{{ LOGOS_ORIGIN }}/img/logos/missing-logo-icon.png"
```

Comment:
would we still be using the `missing-logo-icon` here?

Reply:
If it's in the `/logos/` directory alongside the other breach logos, yup.

Reply:
Well right but I thought we were going back to using the correct image path here then adding a configuration to serve `missing-logo-icon.png` only if the real logo isn't available so that `missing-logo-icon.png` doesn't appear even after we've uploaded the logo.

Reply:
...just noticed the yellow "Outdated" flag above this. I'll be on my way now!